persistence/test/gov_test.go

```diff
@@ -189,5 +189,26 @@ func TestGetSetToggleByteArrayFlag(t *testing.T) {
 
 	require.Equal(t, newOwner, owner)
 	require.Equal(t, true, enabled)
+}
+
+func TestGetAllParams(t *testing.T) {
+	db := NewTestPostgresContext(t, 0)
 
+	err := db.InitGenesisParams(test_artifacts.DefaultParams())
+	require.NoError(t, err)
+
+	appMaxChain, err := db.GetStringParam(AppMaxChainsParamName, 0)
```

Comment:
I think this test involves setting parameters too, I think its best to keep these 3 tests separate as I have opted to keep the `GetStringParam` `GetBytesParam` and `GetIntParam` functions alive so that we can use the generic wrapper `GetParameter` now which is not an interface function.